content/theme/dev-blog-portfolio.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Dev-Blog-Portfolio"
+github: https://github.com/rohitjain00/dev-portfolio-blog
+demo: https://therohitjain.com/
```

Comment:
The demo url cannot be your personal site. It needs to be a demo of the theme using demo content.